package.json

```diff
@@ -93,5 +93,5 @@
     "utf8": "2.1.2",
     "xgettext-js": "3.0.0"
   },
-  "private": true
+  "private": false
```

Comment:
Sorry for the misunderstanding, but I wasn't requesting for this to be changed.  Please revert this, as this isn't meant to be published.

Reply:
Fixed in https://github.com/metabrainz/musicbrainz-server/pull/2377